src/components/pools/deal/DealInformation.tsx

```diff
@@ -96,8 +96,8 @@ export const DealInformation: React.FC<{
           title="Exchange rates"
           tooltip="The ratio at which investment tokens deposited in the pool will be exchanged for deal tokens"
         >
-          <Value>{`${deal.exchangeRates.investmentPerDeal.formatted} ${deal.underlyingToken.symbol} per ${pool.investmentTokenSymbol}`}</Value>
-          <Value>{`${deal.exchangeRates.dealPerInvestment.formatted} ${pool.investmentTokenSymbol} per ${deal.underlyingToken.symbol}`}</Value>
+          <Value>{`${deal.exchangeRates.dealPerInvestment.formatted} ${deal.underlyingToken.symbol} per ${pool.investmentTokenSymbol}`}</Value>
```

Comment:
Maybe add TODO to not forget to switch it again when we will fix origin bug in subgraph?

Reply:
I think the subgraph is okay, the issue was the order

Reply:
Ah, I see, these variable names are little bit confusing 😅